### PR TITLE
Something similar to Google page. Not identical, because I'm too lazy…

### PR DESCRIPTION
**Pull request by okedo**
Something similar to Google page. Not identical, because I'm too lazy… (*^_^*)

### CODE REVIEW

File: homeworks/html-css/Viktor Patraboi/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+</head>
+
+<body>
+    <nav class="navigation">
+        <a href="https://mail.google.com">Gmail</a>
+        <a href="https://www.google.com/imghp?hl=uk&ei=eYw9Wc7BOoq2swHuo7f4Dw&ved=0EKouCAIoAQ">Pictures</a>
+    </nav>
+    <section class="main-wrapper">
+        <div class="logo">
+            <img class="logo-img" src="D:\USR\google.png" title="Google">
```
Comment (denysdovhan) [inline homeworks/html-css/Viktor Patraboi/index.html:20]: Do not use your local absolute paths.

File: homeworks/html-css/Viktor Patraboi/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<html>
```
Comment (denysdovhan) [inline homeworks/html-css/Viktor Patraboi/index.html:1]: Do not use spaces in folder's name

Comment (denysdovhan) [reply homeworks/html-css/Viktor Patraboi/index.html:1]: You had to use your GitHub's username as folder's name, which is @okedo.

**Review by denysdovhan — APPROVED** [review]
LGTM, except the folder's name.

File: homeworks/html-css/Viktor Patraboi/index.html (hunk 1 above)
Comment (okedo) [reply homeworks/html-css/Viktor Patraboi/index.html:20]: resolved

File: homeworks/html-css/Viktor Patraboi/index.html (hunk 2 above)
Comment (okedo) [reply homeworks/html-css/Viktor Patraboi/index.html:1]: resolved

**Pull request merged**